src/ResourceManager/Automation/Commands.Automation.Test/UnitTests/GetAzureAutomationHybridWorkerGroupTest.cs

```diff
@@ -19,7 +21,8 @@ public class GetAzureAutomationHybridWorkerGroupTest : RMTestBase
 
         private GetAzureAutomationHybridWorkerGroup cmdlet;
 
-        [TestInitialize]
+        [Fact]
```

Comment:
@robplank Not quite.  This depends on test execution order, which may vary from test run to test run.  There are two options for test setup in xunit:  Option 1: setup that runs at the start of each test in the class - in this case, just use the constructor, the class constructor for the test class automatically runs at the start of each test.  Option 2: setup that runs only once before any tests in the class are executed.  For this, use the IClassFixtiure pattern.  You can find a good example of this here:  https://github.com/Azure/azure-powershell/blob/83203f0c2ab4988661f570991b4b26a6c8ee8338/src/ResourceManager/KeyVault/Commands.KeyVault.Test/ScenarioTests/KeyVaultManagementTests.cs#L31 

The mechanisms for test seup are described in detail here:  https://xunit.github.io/docs/shared-context.html  if you are interested.